.github/workflows/build-container.yml

```diff
@@ -1,29 +1,38 @@
 name: "CI"
 on:
   push:
-    branches:
-      - devel
   pull_request:
-    branches:
-      - devel
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch:
-env:
-  HUGEPAGES: 256
 
 jobs:
-  build-and-test:
+  build:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
         with:
+          # git submodule update --init --recursive
           submodules: recursive
 
       - name: Build container images
         run: make build SVC="spdk nvmeof nvmeof-cli ceph"
 
+      - name: Save container images
+        run: |
+          . .env
+          docker save quay.io/ceph/nvmeof:$NVMEOF_VERSION > nvmeof.tar
+          docker save quay.io/ceph/nvmeof-cli:$NVMEOF_VERSION > nvmeof-cli.tar
```

Comment:
is there are reason we call the images *.tar?

Reply:
Hello @oritwas, thank you for review 🖖

According to "[docker save](https://docs.docker.com/engine/reference/commandline/save/)":  _Produces a tarred repository to the standard output stream. Contains all parent layers, and all tags + versions, or specified repo:tag, for each argument provided._ 

Their example usage:
```shell
 docker save busybox > busybox.tar
```
So it is natural to call those "*.tar"
WDYT?